tests/functional/c/consider/consider_using_f_string.py

```diff
@@ -35,6 +35,11 @@ def print_good():
     print("%(Param_1)s %(Param_2)s" % PARAM_DICT)
     print("%(Param_1)s %(Param_2)s" % return_dict())
     print("{a[Param_1]}{a[Param_2]}".format(a=PARAM_DICT))
+    print("{}".format("\n"))
+    print("{}".format("\n".join(i for i in "string")))
+    print("%s" % "\n")
+    print("%s" % "\n".join(i for i in "string"))
```

Comment:
```suggestion
    print("{}".format("\n"))  # [consider-using-f-string]
    print("{}".format("\n".join(i for i in "string")))
    print("%s" % "\n")  # [consider-using-f-string]
    print("%s" % "\n".join(i for i in "string"))
```
But maybe that would be another checker, "consider-a-string-without-format" ?

Reply:
? 
See https://stackoverflow.com/questions/44780357/how-to-use-newline-n-in-f-string-to-format-output-in-python-3-6
You can't use \ and thus \n within the expression part. So they shouldn't raise the message.

Reply:
This looked wrong because it can become ``print("\n")`` but that should definitely be another warning "consider-a-string-without-format" .

Reply:
Ah yeah. I only kept it minimal to test both direct string interpolation and interpolation of a function containing backlashes.
There does seem to be room for another checker, but that's something for the future 😃